src/Cool/DidYouKnow/testDidYouKnowModal.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Cool/DidYouKnow/testDidYouKnowModal.hpp"
+#include <fmt/chrono.h>
+#include <imgui.h>
+
+void did_you_know(Cool::DidYouKnowModal& _did_you_know)
```

Comment:
Call this function every frame so that "Did you know" will actually start popping up when we open CoolLab. The right place to call it is in Lab's App class, in its `update()` method.